bok_choy/a11y/axe_core_ruleset.py

```diff
@@ -148,6 +151,53 @@ def set_scope(self, include=None, exclude=None):
 
         self.context = json.dumps(context) if context else 'document'
 
+    def customize_ruleset(self, custom_ruleset_file=None):
+        """
+        Updates the ruleset to include a set of custom rules. These rules will
+        be _added_ to the existing ruleset or replace the existing rule with
+        the same ID.
+
+        Args:
+
+            custom_ruleset_file (optional): The filepath to the custom rules.
+                Defaults to `None`. If `custom_ruleset_file` isn't passed, the
+                environment variable `BOKCHOY_A11Y_CUSTOM_RULES_FILE` will be
+                checked. If a filepath isn't specified by either of these
+                methods, the ruleset will not be updated.
+
+        Raises:
+
+            `IOError` if the specified file does not exist.
+
+        Examples:
+
+            To include the rules defined in `axe-core-custom-rules.js`::
+
+                page.a11y_audit.config.customize_ruleset(
+                    "axe-core-custom-rules.js"
+                )
+
+            Alternatively, use the environment variable `BOKCHOY_A11Y_CUSTOM_RULES_FILE`
+            to specify the path to the file containing the custom rules.
+
+        Documentation for how to write rules:
```

Comment:
@benpatterson 
re: "Do we need to refer anyone to documentation on how to write custom rules?"

This is done here, along with a link to the example in the codebase.